test/dummy/app/components/dummy/ui/menu_toolbar/dropdown_component.js

```diff
@@ -0,0 +1,77 @@
+window.Folio.Stimulus.register('d-ui-menu-toolbar-dropdown', class extends window.Stimulus.Controller {
+  static values = {
+    open: { type: Boolean, default: false },
+    width: Number,
+    dropdownTrigger: String,
+  }
+
+  disconnect () {
+    this.unbindOutsideClick()
+  }
+
+  onAnyClick (e) {
+    const $dropdownTrigger = document.querySelector(`.${this.dropdownTriggerValue}`)
+    
+    if (!this.element.contains(e.target) && !$dropdownTrigger.contains(e.target)) {
+      if (e.type === "keyup" && e.key !== "Enter") return
```

Comment:
Nerozumim tady, proc "Enter". A melo by to asi byt nad tim `if`, ne? Keyup bude mit jako target bud tlacitko pro otevreni dropdown (tedy focus), nebo `body`.